src/main/java/org/humancellatlas/ingest/project/ProjectService.java

```diff
@@ -60,7 +79,20 @@ public Project addProjectToSubmissionEnvelope(SubmissionEnvelope submissionEnvel
                                      });
     }
 
-    public Page<Project> queryByContent(List<MetadataCriteria> query, Pageable pageable){
-        return this.projectRepository.findByContent(query, pageable);
+    public Page<Project> findByCriteria(List<MetadataCriteria> criteriaList, Boolean andCriteria, Pageable pageable){
+        return this.projectRepository.findByCriteria(criteriaList, andCriteria, pageable);
+    }
+
+    public Page<SubmissionEnvelope> getProjectSubmissionEnvelopes(Project project, Pageable pageable) {
```

Comment:
Since the pageable param isn't used here, can it be removed?

Reply:
it is actually used on the return object.